torchtune/training/quantization.py

```diff
@@ -43,6 +48,22 @@
         Int8DynActInt4WeightQATQuantizer,
     )
 
+from torchtune.utils._version import torch_version_ge
+
+
+_SUPPORTS_INT8_MIXED_PRECISION_TRAINING = (
+    torch_version_ge("2.4.0")
+    and Version(torchao.__version__) >= Version("0.7")
```

Comment:
this didnt work for me

![image](https://github.com/user-attachments/assets/5eb214f8-4608-4779-b756-74be010fbb77)

Reply:
Apparently I need to add `.dev` https://github.com/pytorch/ao/blob/2ba1a61fe1244560325b5051b5d3c10044553be0/torchao/utils.py#L529-L532

I was installing torchao from source, which has version number as `0.7.0+gitxxxxx`, and the comparison works as expected. Will push the change.